### ISSUE TITLE
Comments // stylelint-disable-line and /* prettier-ignore */ are ignored

### ISSUE THREAD

**Issue by elkinuk** [hugomrdias/prettier-stylelint#27]
After starting prettier-stylelint --write it is expected, that lines, marked like disabled or ignored, wouldn't be changed. But they are changing. 

P.S: To .stylelintrc was added

```
"extends": "./node_modules/prettier-stylelint/config.js"
```

Everything was done like in the tutorial

**Comment by isellsoap** [comment]
@elkinny I digged a bit into it and found out that the underlying issue is not caused by this repository, but by stylelint itself. stylelint doesn’t respect `/* stylelint-disable */` comments when doing the autofix operation. The issue where this is tracked is https://github.com/stylelint/stylelint/issues/2643. The latest update with the issue is this comment: https://github.com/stylelint/stylelint/issues/3867#issuecomment-450949484. 

Long story short: `// prettier-ignore` is being respected, but as the stylelint autofix command is being executed as the last command this doesn’t help (as pointed out above).
